docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,90 +10,82 @@ keywords: changelog, table, materialize
 
 ## Context
 
-You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog stream into a table. This is broadly called *materializing* a changelog stream into a table.
+You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table. This is broadly called *materializing* a changelog into a table.
 
 ## In action
 
 ```sql
 CREATE TABLE t1 AS
-    SELECT k,
-           LATEST_BY_OFFSET(v1) AS v1,
-           LATEST_BY_OFFSET(v2) AS v2,
-           LATEST_BY_OFFSET(v3) AS v3
-    FROM s1
-    GROUP BY k
+    SELECT *
+    FROM changelog
     EMIT CHANGES;
 ```
 
-## Materializing a changelog stream
+## Materializing a changelog
 
-In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, use the `LATEST_BY_OFFSET` aggregation.
+In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, declare the `changelog` as a table.
```

Comment:
this is no longer accurate - you don't need to `aggregate other streams and tables`

Reply:
Not sure what you mean @agavra? To "convert" a STREAM into a TABLE you can only do it with an aggregation.

Of course, a table-table join also gives you a result table. Thus the sentence is still misleading.